app/controllers/courses_controller.rb

```diff
@@ -1926,7 +1926,10 @@ def copy_course
   #   - 'public_domain' (Public Domain).
   #
   # @argument course[is_public] [Boolean]
-  #   Set to true if course if public.
+  #   Set to true if course is public.
+  #
+  # @argument course[is_public_to_auth_users] [Boolean]
```

Comment:
if we're going to document this, we should return it in the API response for courses, and clarify that is_public makes it public to both authenticated and non-authenticated users